MiniApp/Classes/RealMiniApp.swift

```diff
@@ -146,6 +146,17 @@ internal class RealMiniApp {
     func getDownloadedListWithCustomPermissions() -> MASDKDownloadedListPermissionsPair {
         return self.miniAppStatus.getMiniAppsListWithCustomPermissionsInfo() ?? []
     }
+    
+    func createMASDKErrorCompletionHandler<T>(completionHandler: @escaping (Result<T, MASDKError>) -> Void) -> (Result<T, Error>) -> Void {
```

Comment:
Should we rename it to some other name? Because it handles both success and failure completion handlers.